support/rewriteInheritance.js

```diff
@@ -0,0 +1,29 @@
+var through = require('through2');
+
+/**
+ * Replace all instances of `__proto__` in a string with a cross-browser
+ * compatible alternative.
+ *
+ * @param {string} js
+ * @return {string}
+ */
+function parseInheritance(js) {
+  return js.replace(/^ *(\w+)\.prototype\.__proto__ * = *(\w+)\.prototype *;?/gm, function(_, child, parent){
```

Comment:
errr... yeah.  we need to get rid of all of these hinky shims and use something well-supported.  if we're targeting environments that don't support `Object.create()` then we should just do something like use [lodash.create](https://www.npmjs.com/package/lodash.create)